gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -28,6 +28,7 @@
 #include "TROOT.h"
 #include "TEnv.h"
 #include "TExec.h"
+#include "TSocket.h"
```

Comment:
That makes the build failing on Windows. And this requires the following change:
```
diff --git a/gui/webdisplay/CMakeLists.txt b/gui/webdisplay/CMakeLists.txt
index 50542b3ff6..1f68a838dd 100644
--- a/gui/webdisplay/CMakeLists.txt
+++ b/gui/webdisplay/CMakeLists.txt
@@ -23,6 +23,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTWebDisplay
        Core
        RHTTP
        MathCore
+       Net
 )

 ROOT_ADD_TEST_SUBDIRECTORY(test)
```

Reply:
Sorry, please see #11824